projects/Makefile.MIYOO

```diff
@@ -1,16 +1,15 @@
 DEVKIT = /opt/miyoomini-toolchain/
 CROSS_COMPILE=$(DEVKIT)bin/arm-linux-gnueabihf-
-# CROSS_COMPILE=$(DEVKIT)bin/arm-buildroot-linux-musleabi-
 
 CC = $(CROSS_COMPILE)gcc
 CXX = $(CROSS_COMPILE)g++
 SYSROOT     := $(shell $(CROSS_COMPILE)gcc --print-sysroot)
 SDL_CFLAGS  := $(shell $(SYSROOT)/usr/bin/sdl-config --cflags)
 SDL_LIBS    := $(shell $(SYSROOT)/usr/bin/sdl-config --libs)
 
-DEFINES = -DHAVE_STDINT_H -DVERSION_BITTBOY
+DEFINES = -DHAVE_STDINT_H -DVERSION_BITTBOY -DPLATFORM_DINGOO -D_NDEBUG -D_NO_JACK_ -I$(PWD)/../sources -DCPP_MEMORY $(SDL_CFLAGS) $(SDL_LIBS) -DBUFFERED
 INCLUDES = -Iinclude $(SDL_CFLAGS)
-OPT_FLAGS  = -Ofast -fdata-sections -fdata-sections -fno-common -fno-PIC -flto 
+OPT_FLAGS  = -O3 -Ofast -fdata-sections -fdata-sections -fno-common -fno-PIC -flto -marm -mtune=cortex-a7 -mfpu=neon-vfpv4 -mfloat-abi=hard -march=armv7ve+simd
```

Comment:
Cool! I got an issue of choppyness on an RG unit, maybe something similar to this might tight that up?